src/frontend/web_application/src/scenes/DiscussionList/index.js

```diff
@@ -0,0 +1,25 @@
+import { createSelector } from 'reselect';
+import { bindActionCreators, compose } from 'redux';
+import { connect } from 'react-redux';
+import { withTranslator } from '@gandi/react-translate';
+import { requestDiscussions, loadMoreDiscussions, hasMore } from '../../store/modules/discussion';
+import Presenter from './presenter';
+
+const discussionSelector = state => state.discussion;
+const mapStateToProps = createSelector(
+  [discussionSelector],
+  state => ({
+    discussions: state.discussions.map(discussionId => state.discussionsById[discussionId]),
```

Comment:
It isn't already one ?

I probably need to refactor all my selectors for correct memoization. It's in my todo list.